### DISCUSSION TITLE
trying to deploy with fontawesome pro as a package.

### DISCUSSION THREAD

**Discussion by jamitz440** [FortAwesome/Font-Awesome#20268] [Q&A]
I have an .npmrc file set up like so:

```
@awesome.me:registry=https://npm.fontawesome.com/
@fortawesome:registry=https://npm.fontawesome.com/
//npm.fontawesome.com/:_authToken=${NPM_AUTH}
```

and a NPM_AUTH environment variable. 

But it fails on build. I am quite new to self hosted deployments and cant seem to figure this one out. 

anyone know a possible solution for this?

**Comment by robmadole** [comment]
@jamitz440 send an email to hello@fontawesome.com. Since this is a Pro account we'll be able to help you better there in private.
